src/pages/use-cases/code-health.tsx

```diff
@@ -238,30 +238,27 @@ const UseCasePage: FunctionComponent = () => (
                                     entire codebase.
                                 </div>
                                 <div className="d-flex flex-column flex-lg-row pt-1">
-                                    <Link
-                                        href="/demo"
-                                        passHref={true}
-                                        data-button-style={buttonStyle.primary}
-                                        data-button-location={buttonLocation.hero}
-                                        data-button-type="cta"
-                                    >
+                                    <Link href="/demo" passHref={true}>
                                         {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
                                         <a
                                             className="btn btn-primary mr-lg-3 mb-lg-0 mb-3 w-md-100"
                                             title="Request a Demo."
+                                            data-button-style={buttonStyle.primary}
```

Comment:
Thank you for doing this! I also happened to update these Use Case page instances in response to a [recent comment ](https://github.com/sourcegraph/about/pull/5365#discussion_r881673706) in #5365. I will reconcile them in case there are merge conflicts. Reference: https://github.com/sourcegraph/about/commit/235509883f01ddbcdae881cb6593885c1af8fc28